openmetadata-service/src/main/java/org/openmetadata/service/apps/ApplicationHandler.java

```diff
@@ -144,9 +145,17 @@ public void migrateQuartzConfig(App application) throws SchedulerException {
       return;
     }
     LOG.info("migrating app quartz configuration for {}", application.getName());
+    @SuppressWarnings("unchecked")
+    Map<String, Object> map = JsonUtils.readValue(appInfo, Map.class);
+    if (map.containsKey("appSchedule")) {
```

Comment:
@sushi30 any reason this migration will not able to take care of it? https://github.com/open-metadata/OpenMetadata/blob/84941a7d32d1d9c630f267b3e4fbdc19279dfbc8/bootstrap/sql/migrations/native/1.3.2/mysql/schemaChanges.sql#L5

Reply:
@harshach `QUARTZ_JOB_DETALS.JOB_DATA` holds java serialized data. There is not elegant way to manipulate it with SQL. this is one of the main motivations for removing all job context from the quartz tables.